db/schema.rb

```diff
@@ -26,19 +26,30 @@
 
   create_table "items", force: :cascade do |t|
     t.decimal "price"
-    t.integer "stock_level"
+    t.integer "status"
     t.string "image"
     t.string "name"
     t.text "description"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
+  create_table "order_items", force: :cascade do |t|
+    t.bigint "item_id"
+    t.bigint "order_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "quantity"
+    t.index ["item_id"], name: "index_order_items_on_item_id"
+    t.index ["order_id"], name: "index_order_items_on_order_id"
+  end
+
   create_table "orders", force: :cascade do |t|
     t.bigint "user_id"
     t.integer "status", default: 0
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.float "total_price"
```

Comment:
Same here.

Reply:
I think total price is fine here. How else would we be able to call total price for that order? Unless that's not something we need to have as a column and instead just an instance method that can be called in the view?